### PR TITLE
MAILET-106 Refactor RemoveMimeHeader

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: mailet/standard/src/test/java/org/apache/james/transport/mailets/RemoveMimeHeaderTest.java [hunk 1]
```diff
@@ -20,107 +20,143 @@
...
-    private final static String HEADER2 = "header2";
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
 
-    private GenericMailet setupMockedMailet(String name1, String name2) throws MessagingException {
-        GenericMailet mailet = new RemoveMimeHeader();
-        FakeMailetConfig mci = new FakeMailetConfig("Test", new FakeMailContext());
-        if (name1 != null) mci.setProperty("name", name1);
-        if (name2 != null) mci.setProperty("name", name2);
+    private GenericMailet mailet;
```
Comment (rouazana) [inline mailet/standard/src/test/java/org/apache/james/transport/mailets/RemoveMimeHeaderTest.java]: testee/sut?

**Comment by mbaechler** [comment]
👍

**Comment by rouazana** [comment]
:+1:

**Comment by chibenwa** [comment]
:+1:

**Pull request merged**